aspnetcore/signalr/hubs.md

```diff
@@ -609,6 +612,76 @@ Using `Hub<IChatClient>` enables compile-time checking of the client methods. Th
 > [!NOTE]
 > The `Async` suffix isn't stripped from method names. Unless a client method is defined with `.on('MyMethodAsync')`, don't use `MyMethodAsync` as the name.
 
+## Client results
+
+In addition to making calls to clients, the server can request a result from the client. This requires the server to use `ISingleClientProxy.InvokeAsync` and the client to return a result from their `.On` handler.
+
+There are two ways to use the API on the server, the first is to call `Single(...)` on the `Clients` property in a Hub method.
+```csharp
+public class ChatHub : Hub
+{
+    public async Task<string> WaitForMessage(string connectionId)
+    {
+        var message = await Clients.Single(connectionId).InvokeAsync<string>("GetMessage");
+        return message;
+    }
+}
+```
+
+> [!NOTE]
+> Currently using `InvokeAsync` from a Hub method requires setting the [MaximumParallelInvocationsPerClient](xref:signalr/configuration#configure-server-options) option to a value greater than 1.
+
+The second way is to call `Single(...)` on an instance of [`IHubContext<T>`](xref:signalr/hubcontext).
+```csharp
+async Task SomeMethod(IHubContext<MyHub> context)
+{
+    string result = await context.Single(connectionID).InvokeAsync<string>("GetMessage");
+}
+```
+
+Strongly-typed hubs can also return values from interface methods:
+```csharp
+public interface IClient
+{
+    Task<string> GetMessage();
+}
+
+public class ChatHub : Hub<IClient>
+{
+    public async Task<string> WaitForMessage(string connectionId)
+    {
+        string message = await Clients.Single(connectionId).GetMessage();
+        return message;
+    }
+}
+```
+
+Clients return results in their `.On(...)` handlers as shown below:
+
+#### .NET client
+```csharp
+hubConnection.On("GetMessage", async () =>
+{
+    Console.WriteLine("Enter message:");
+    var message = await Console.In.ReadLineAsync();
+    return message;
+});
+```
+#### Typescript client
+```typescript
+hubConnection.on("GetMessage", async () => {
+    let promise = new Promise((resolve, reject) => {
+        setTimeout(() => {
+            resolve("message");
+        }, 100);
+    });
+    return promise;
+});
+```
+
+> [!NOTE]
+> Client results do not work with the Azure SignalR Service currently.
```

Comment:
Another case where "currently" is best tracked by an issue, and the text just conveys the info to the reader.

```suggestion
> Client results don't work with the Azure SignalR Service.
```

On this one (and the earlier "currently" case), I sometimes go with something like, 'This will be addressed in a future release.' Sometimes, I cross-link the PU issue or the docs issue, 'For more information, see [Blah blah blah (dotnet/aspnetcore #12345](... LINK ...).'